resources/assets/js/vues/components/dropzone.js

```diff
@@ -13,8 +13,9 @@ function mounted() {
    let container = this.$el;
    let _this = this;
    this._dz = new DropZone(container, {
-	addRemoveLinks: true,
-	dictRemoveFile: trans('components.image_upload_remove'),
+        addRemoveLinks: true,
+        dictRemoveFile: trans('components.image_upload_remove'),
+        timeout: +window.dropZoneTimeout || 60000,
```

Comment:
The default timeout could be 0 (disabled), with this, the server keep with the responsibility of cancel the request. And if the website need of timeout, can use the custom `window.dropZoneTimeout`.
What you think?